src/lando/main/scm/hg.py

```diff
@@ -356,11 +361,7 @@ def format_stack_amend(self) -> Optional[list[str]]:
             raise exc
 
     def format_stack_tip(self, commit_message: str) -> Optional[list[str]]:
-        """Add an autoformat commit to the top of the patch stack.
-
-        Returns a list containing a single string representing the ID of the newly created commit.
```

Comment:
May be removed by accident? Either way, no biggie.

Reply:
I did it on purpose: I kept the one-line description here, and have the full doc-string in the abstract SCM, so we have a single source of truth for the details.